### ISSUE TITLE
train yout own dataset

### ISSUE THREAD

**Issue by AliDs2022** [k2-fsa/sherpa#246]
hello , i have persian dataset. 
can i have train models with this dataset????

**Comment by csukuangfj** [comment]
Yes, please have a look at https://github.com/k2-fsa/icefall/issues/740

Also, we have a recipe for Arabic. Please see https://github.com/k2-fsa/icefall/tree/master/egs/mgb2/ASR

You can also try the pre-trained models trained with Arabic dataset at
https://huggingface.co/spaces/k2-fsa/automatic-speech-recognition
in your browser without installing anything.
